### ISSUE TITLE
DAC_Nontext_Contrast_01_Updated Issue

### ISSUE THREAD

**Issue by sarah-beasley** [open-sdg/open-sdg#1841]
URL: http://uk-sdg-feature-branches.s3-website.eu-west-2.amazonaws.com/feature-2.1.0-beta2/reporting-status/

Issue raised in DAC report - The yellow (#ffdd00) border used to indicate when the links are focused fails colour contrast guidelines with a ratio of 1.3:1 against the white background.

![image](https://user-images.githubusercontent.com/106670682/197177387-1e41e796-7cfa-41bf-be33-64a9b771e386.png)

We want all goal icons which are used as links to be styled the same as the goal icon links on the homepage (i.e. with black border then yellow border). Please can you add the black border to the goal icon links on the reporting status page.
